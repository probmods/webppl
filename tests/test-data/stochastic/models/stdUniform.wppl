var model = function() {
  return uniform();
};
