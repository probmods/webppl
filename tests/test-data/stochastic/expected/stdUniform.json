{
  "mean": 0.5,
  "std": 0.288675
}
